website/client/src/components/admin-panel/index.vue

```diff
@@ -0,0 +1,754 @@
+                      and have not yet retamed it.
+                    </p>
+                    <p v-if="itemType === 'special'">
+                      When there are 0 of these items, we can't tell if
+                      they had been owned and were all used, or have never been owned.
+                    </p>
+                    <p v-if="itemType === 'gear'">
+                      A value of true means they own the item now and can wear it.
+                      A value of false means they used to own it but lost it from Death
+                      (or an old Rebirth).
+                    </p>
+
+                    <ul>
+                      <li
+                        v-for="item in collatedItemData[itemType]"
+                        :key="item.key"
+                        class="form-group form-inline"
+                      >
+                        <button
+                          class="form-control btn btn-primary"
+                          @click="changeData(item.path, item.value)"
+                        >
+                          Change
+                        </button>
+                        <span>{{ item.valueForDisplay }} : </span>
+                        <span v-bind:class="{ ownedItem: !item.neverOwned }">{{ item.key }} :</span>
+                        <span v-html="item.name"></span>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="accordion">
+            <div
+              class="accordion-group"
+              heading="Update Items"
+            >
+              <h3
+                class="expand-toggle"
+                :class="{'open': expandUpdateItems}"
+                @click="expandUpdateItems = !expandUpdateItems"
+              >
+                Update Items
+              </h3>
+              <div
+                v-if="expandUpdateItems"
+                class="form-group well"
+              >
+                <p>
+                  TIP: First find the item in the "Items" section above and
+                  click its "Change" button.
+                  The item path and current value (if any) will be inserted below.
+                </p>
+                <input
+                  v-model="hero.itemPath"
+                  class="form-control"
+                  type="text"
+                >
+                <small class="muted">
+                  Enter the <strong>item path</strong>. E.g.,
+                  <code>items.pets.BearCub-Zombie</code> or
+                  <code>items.gear.owned.head_special_0</code> or
+                  <code>items.gear.equipped.head</code>.
+                  You can find all the items in the "Item" section above.
+                  The "Change" buttons will insert the path and the current value automatically.
+                </small>
+                <br>
+                <input
```

Comment:
this should be a form

Reply:
Done in 26fdcbbee51937d39a149a20bf7ccd1cf674ca18